packages/next/src/client/components/router-reducer/fetch-server-response.ts

```diff
@@ -84,6 +87,10 @@ export async function fetchServerResponse(
     headers[NEXT_ROUTER_PREFETCH_HEADER] = '1'
   }
 
+  if (isFastRefresh) {
```

Comment:
should we wrap this in `process.env.NODE_ENV === 'development'`?

Reply:
Oh yes, good point, wanted to that initially and then forgot about it. Will fix...